src/content-helper/dashboard-widget/dashboard-widget.tsx

```diff
@@ -12,12 +12,15 @@ import { TopPosts } from './components/top-posts';
 window.addEventListener(
 	'load',
 	function() {
-		render(
-			<VerifyCredentials>
-				<TopPosts />
-			</VerifyCredentials>,
-			document.querySelector( '#wp-parsely-dashboard-widget > .inside' )
-		);
+		const container = document.querySelector( '#wp-parsely-dashboard-widget > .inside' );
+		if ( null !== container ) {
+			const root = createRoot( container );
```

Comment:
This change specifically is addressing a deprecated warning that was being thrown in the browser console.